cmd/crc/cmd/setup_test.go

```diff
@@ -14,7 +14,7 @@ func TestSetupRenderActionPlainSuccess(t *testing.T) {
 	assert.NoError(t, render(&setupResult{
 		Success: true,
 	}, out, ""))
-	assert.Equal(t, "Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster\n", out.String())
+	assert.Equal(t, "Your system is correctly setup for using CodeReady Containers, run 'crc start' to start the instance\n", out.String())
```

Comment:
Still incorrect

"Your system is correctly setup for using CodeReady Containers. Run 'crc start' to start the instance"